.travis.yml

```diff
@@ -35,6 +32,10 @@ matrix:
     - python: 3.5
       env: TOXENV=flake8
     - python: 3.5
+      env: TOXENV=coverage
+      addons:
+        firefox: latest
+    - python: 3.5
```

Comment:
I think I had a conflict with my travis file and the one on master, so I had just took the one on master.